diracx-core/src/diracx/core/models.py

```diff
@@ -272,3 +272,18 @@ class JobCommand(BaseModel):
     job_id: int
     command: Literal["Kill"]
     arguments: str | None = None
+
```

Comment:
You should add `PilotStatus` similar to what is done for `JobStatus`, and use later on.